mern/client/src/Interactor/recoverSessionInteractor.js

```diff
@@ -0,0 +1,18 @@
+import { getFlowBySessionID } from "../DAO/recoverSessionDAO"
+//  import dao to call axios function
+// manipulate data
+
+// sessionID does not need to be manipulated so this function sends it straight to DAO
+export async function sendToDAO(sessionID) {
+    // returns a flow promise object
+    const flow = await getFlowBySessionID(sessionID);
+    console.log("this is the flow", flow);
+    return flow;
```

Comment:
Keep in mind that the flow object is a promise, so you would need to access the contents with a .then

Reply:
*flow object is a json object, not a promise. Comments have been updated to reflect that.